jobs/scheduler/templates/application.properties.erb

```diff
@@ -38,6 +38,9 @@ client.ssl.trust-store=/var/vcap/data/certs/scalingengine/cacerts
 client.ssl.trust-store-password=123456
 client.ssl.protocol=TLSv1.2
 
+#Quartz
+org.quartz.scheduler.instanceName=app-autoscaler
```

Comment:
For cluster, should use same name(org.quartz.scheduler.instanceName).

org.quartz.scheduler.instanceName

Can be any string, and the value has no meaning to the scheduler itself - but rather serves as a mechanism for client code to distinguish schedulers when multiple instances are used within the same program. If you are using the clustering features, you must use the same name for every instance in the cluster that is ‘logically’ the same Scheduler.

http://www.quartz-scheduler.org/documentation/quartz-2.x/configuration/ConfigMain.html

The instance id inside the application context is unique which is set to Auto
 <prop key="org.quartz.scheduler.instanceId">AUTO</prop>